bitnami/postgresql-ha/templates/_helpers.tpl

```diff
@@ -48,6 +48,21 @@ Fully qualified app name for LDAP
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return the proper Node hostname
+*/}}
+{{- define "postgresql-ha.nodesHostnames" -}}
+{{- $postgresqlReplicaCount := int .Values.postgresql.replicaCount }}
+{{- $postgresqlFullname := include "postgresql-ha.postgresql" $ }}
+{{- $postgresqlHeadlessServiceName := printf "%s-headless" (include "postgresql-ha.postgresql" $) }}
+{{- $releaseNamespace := .Release.Namespace }}
+{{- $clusterDomain:= .Values.clusterDomain }}
+{{- range $e, $i := until $postgresqlReplicaCount -}}
+{{- $nodeHostname := printf "%s-%d.%s.%s.svc.%s" $postgresqlFullname $i $postgresqlHeadlessServiceName $releaseNamespace $clusterDomain }}
+{{- printf "%d:%s:5432," $i ($nodeHostname | trunc 128 | trimSuffix "-" | trimSuffix ".") -}}
+{{- end -}}
```

Comment:
This code will generate as string that ends in a comma `","` regardless the number or replicas. Alternative: use a list + `join ","`:
```suggestion
{{- $nodeList := list }}
{{- range $e, $i := until $postgresqlReplicaCount -}}
{{- $nodeHostname := printf "%s-%d.%s.%s.svc.%s" $postgresqlFullname $i $postgresqlHeadlessServiceName $releaseNamespace $clusterDomain }}
{{- $nodeList = append $nodeList (printf "%d:%s:5432" $i ($nodeHostname | trunc 128 | trimSuffix "-" | trimSuffix ".")) }}
{{- end -}}
{{ join "\n" $nodeList }}
```

Reply:
I didn't want to change the current behavior, as the current string is always ending in `,` I have left that as it is. But if it doesn't matter I can use your approach removing the final `,`